specification/applicationinsights/resource-manager/Microsoft.Insights/stable/2021-08-01/examples/WorkbookRevisionGet.json

```diff
@@ -0,0 +1,35 @@
+{
+  "parameters": {
+    "api-version": "2021-08-01",
+    "subscriptionId": "6b643656-33eb-422f-aee8-3ac145d124af",
+    "resourceGroupName": "my-resource-group",
+    "resourceName": "deadb33f-5e0d-4064-8ebb-1a4ed0313eb2",
+    "revisionId": "1e2f8435b98248febee70c64ac22e1ab"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "c0deea5e-3344-40f2-96f8-6f8e1c3b5722",
```

Comment:
>"c0deea5e-3344-40f2-96f8-6f8e1c3b5722", [](http://example.com/codeflow?start=14&length=39)

resource's id property should be the ARM resource id

/subscriptions/<subscriptionid>/etc....